dbms/src/Storages/Page/V3/WAL/serialize.cpp

```diff
@@ -152,7 +152,7 @@ void deserializePutExternalFrom([[maybe_unused]] const EditRecordType record_typ
     assert(record_type == EditRecordType::PUT_EXTERNAL || record_type == EditRecordType::VAR_EXTERNAL);
 
     PageEntriesEdit::EditRecord rec;
-    rec.type = EditRecordType::PUT_EXTERNAL;
+    rec.type = record_type;
```

Comment:
also remove this line
https://github.com/pingcap/tiflash/blob/2cb02ae11882b8fcaf768f38642d53b113f20995/dbms/src/Storages/Page/V3/WAL/serialize.cpp#L103

Reply:
Add related ut for VAR_XXX in `WALSeriTest` in `dbms/src/Storages/Page/V3/tests/gtest_wal_store.cpp`

Reply:
done